code/modules/power/smes_construction.dm

```diff
@@ -41,23 +41,23 @@
 
 // These are used on individual outposts as backup should power line be cut, or engineering outpost lost power.
 // 1M Charge, 150K I/O
-/obj/machinery/power/smes/buildable/outpost_substation/New()
-	..(0)
+/obj/machinery/power/smes/buildable/outpost_substation/Initialize()
+	. = ..()
 	component_parts += new /obj/item/weapon/smes_coil/weak(src)
```

Comment:
Can these not use default_add_parts?

Reply:
In this case its not quite that simple, it require creating special circuit board types for these mapped-only smes types.  Best way might be to adopt something along the lines of the `additional_spawn_components` Baystation has, but that sort of revamp would be its own project I think.